apps/web/vibes/soul/examples/card.tsx

```diff
@@ -4,6 +4,7 @@ export default function Preview() {
   return (
     <div className="flex min-h-48 items-center justify-center p-5 @container">
       <Card
+        id="1"
```

Comment:
This prop shouldn't be in `CardProps`, only when there's a list. In those cases, it's likely that it's not a "card" that will have an ID but the thing being mapped on. For example, maybe there's a mapping over products so the product ID can be used as a key for each card